src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +20,6 @@ public static void main(String[] args) {
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
```

Comment:
Be aware that CompletableFuture.allOf does not handle exceptions by default. If any of the CompletableFutures completed exceptionally, the resulting CompletableFuture returned by allOf will also complete exceptionally. Consider handling possible exceptions here.